Xamarin.Android-Tests.sln

```diff
@@ -96,6 +96,12 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Java.Interop-Tests", "src\M
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Java.Interop", "external\Java.Interop\src\Java.Interop\Java.Interop.csproj", "{94BD81F7-B06F-4295-9636-F8A3B6BDC762}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "tests", "tests", "{AB21EC70-39C8-4042-A4A7-8DCFED863168}"
```

Comment:
Not sure why we're adding new `tests` and `emulator` "projects" (folders?).

Reply:
Thats quite interesting, vscode seems to add the folders as project reference when you use the IDE to add a project to the Solution. I have manually fixed it.